flexwrfoutput/openfiles.py

```diff
@@ -0,0 +1,64 @@
+"""
+Functions to handle raw netCDF output of FLEXPART-WRF.
+
+Meant to be used before the postprocessing with the accessor.
+"""
+from pathlib import Path
+from typing import Tuple, Union
+
+import xarray as xr
+
+
+def combine(flxout: xr.Dataset, header: xr.Dataset) -> xr.Dataset:
+    """Combines dimensions of flxout with header to have full information of output in\
+        one xarray.
+
+    Args:
+        flxout (xr.Dataset): Loaded flxout file.
+        header (xr.Dataset): Loader header file.
+
+    Returns:
+        xr.Dataset: Combined Dataset
+    """
+    combined = xr.merge([flxout, header.drop_dims("Time")])
+    return combined
+
+
+def get_output_paths(path: Union[str, Path]) -> Tuple[Path, Path]:
+    """Finds header and flxout files in directory and returns their paths.
+
+    Args:
+        path (Union[str, Path]): Path of output directory of FLEXPART-WRF.
+
+    Returns:
+        Tuple[Path, Path]: (flxout path, header path)
+    """
+    path = Path(path)
+    header_files = [file for file in path.iterdir() if "header" in str(file)]
+    flxout_files = [file for file in path.iterdir() if "flxout" in str(file)]
```

Comment:
Ich muss mal drüber nachdenken wie man das noch schöner machen kann. Ich finde es nicht super elegant, dass man das in eine Liste schreibt und dann einfach das erste Element nimmt....

Reply:
Habe es etwas umgeschrieben mit `Path.glob` aber an sich bliebt es von der Funktionsweise gleich. Die `len` Eigenschaft brauche ich leider schon um sicher zu gehen, dass das Geladene eindeutig ist.